lang/udf_cache.hh

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "cql3/functions/function_name.hh"
```

Comment:
I'ts a wasm cache, not a udf cache.

Reply:
I changed it to wasm instance cache, hopefully not too verbose

Reply:
verbose = better.